crates/iota/src/validator_commands.rs

```diff
@@ -380,9 +380,10 @@ impl IotaValidatorCommand {
             }
 
             IotaValidatorCommand::LeaveValidators { gas_budget } => {
-                // Only an active validator can leave committee.
+                // Only a committee member can leave the committee.
```

Comment:
This is not correct, it is not about leaving the committee, its about leaving the whole list, so both Active and Committee are accepted here

Reply:
You're right, got it wrong because of the wrong comment, fixed in https://github.com/iotaledger/iota/pull/6381/commits/cc4a90c15c7e45a40ebb248acb9691d2af17f96f